### DISCUSSION TITLE
Make the daily challenge counter appear in the Lazer embedded website

### DISCUSSION THREAD

**Discussion by Shavixinio** [ppy/osu#29358] [Ideas] [answered]
Currently, you can't see the daily challenge counter in the Lazer embedded website.
Screenshot from Lazer:
![Zrzut ekranu 2024-08-09 012122](https://github.com/user-attachments/assets/b937c45c-c8a5-407e-8871-35f5946fd3a9)
Screenshot from the website:
![Zrzut ekranu 2024-08-09 012443](https://github.com/user-attachments/assets/9afa5c39-89f2-494b-a4d7-b74c730917df)

**Comment by peppy** [comment, marked as answer]
It's already added in the next release.
